source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -192,6 +211,7 @@ void CacheFilter::getBody() {
       }
     });
   });
+  ongoing_fetch_ = true;
```

Comment:
While it doesn't actually matter with siloed execution, it's more natural and more obviously correct to first set ongoing_fetch_, then call lookup_->getBody. The current order will look to some like a possible race.